src/main/java/com/google/api/gax/grpc/ApiCallSettingsTyped.java

```diff
@@ -1,13 +1,23 @@
 package com.google.api.gax.grpc;
 
+import com.google.api.gax.core.RetrySettings;
+
 import io.grpc.MethodDescriptor;
+import io.grpc.Status;
+
+import java.util.Set;
 
 /**
```

Comment:
I change the constructor to public since the whole class is package-private. It would be redundant to have a package-private constructor again.